economist-components/user
# Basic version for initial user information

I need this initial prototype for the Omniture plugin but obviously should be a reusable class to get the information regarding the user for the cookie.

To test it add this row to your host file:

127.0.0.1 blog.economist.com

Open 2 different browsers and in one log-in into the economist website and do not log-in in the second one.
After running the example open the bowser at blog.economist.com:<PortUsedByTheLocalServer> and see the difference.

I keep the tech debt to add test.
